activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -93,6 +99,34 @@ def travel(duration, &block)
       #   end
       #   Time.current # => Sat, 09 Nov 2013 15:34:49 EST -05:00
       def travel_to(date_or_time)
+        if block_given? && simple_stubs.stubbed?(Time, :now)
+          travel_to_nested_block_call = <<-MSG
+
+      Calling `travel_to` with a block, when we have previously already made a call to `travel_to`, can lead to confusing time stubbing.
+
+      Instead of:
+
+         travel_to 2.days.from_now do
+           # 2 days from today
+           travel_to 3.days.from_now do
+             # 5 days from today
+           end
+         end
+
+      preferred way to achieve above is:
+
+         travel_to 2.days.from_now do
+           # 2 days from today
+         end
+
+         travel_to 5.days.from_now do
```

Comment:
Thinking about this some more it's weird to use `travel_to` with a relative time. Shouldn't we say `travel 5.days`?

Also actual `travel` calls would spout these exceptions mentioning `travel_to` — that's leaking the implementation to users.